globalmangrovewatch/gmw-users
# Section 6: 6.2c Custom Tabular Input Group

# Description

- created a custom tabular input group for 6.2c as it has more fields and different inputs (selects vs text).  Added a vertical display. Maybe we can get Al's input on whether they should just all be vertical rows for consistency.
- Not sure if I like the design of where the delete icon is, open to other suggestions
- Fixed a bug in section 5 tabular input group where add-unit-input wasn't showing
- todo: create separate task to add modal pop up for deleting items in tabular input to avoid accidental deletions
- I updated the question order. **6.2d** is now **6.2c**, since in the previous question I combined 6.2b/c from the original order, and just called it 6.2b. 


Testing
- create a new site to ensure data is clean
- go to section 6
- scroll down to 6.2c
- test adding new measurement rows
- if purpose is 'Other', additional input should appear to specify category
- test editing/updating data and refreshing the page to ensure data is correct

Fixes # ([80](https://github.com/globalmangrovewatch/gmw-users/issues/80))
